src/main/java/com/modus/camel/datasonnet/language/model/DatasonnetExpression.java

```diff
@@ -1,87 +0,0 @@
-package com.modus.camel.datasonnet.language.model;
```

Comment:
Is there any reason why model is removed?

Reply:
I should've wrote down the reason when I did it... If I remember correctly this class was throwing some errors when I was trying to run it with camel 330, and I couldn't find another camel component that used something similar, so I thought maybe it's something that camel itself had changed in its use? When I removed it and added the Expression constructor to the other DatasonnetExpression all tests passed so I went with it.

Reply:
Not being a convention anywhere else is a good reason to do it alone, so I'm good with this change

Reply:
See my comment below, I think we should keep this class even though it's not used in our code.